cloudevents/extensions/jwt.md

```diff
@@ -0,0 +1,147 @@
+# JWT Extension
+
+This extension provides a way to map JWT (JSON Web Token) values in the formats of JWE
+ (JSON Web Encryption) or JWS (JSON Web Signature).
+ 
+JWE MAY be used to encrypt a CloudEvent data and JWS MAY be used to sign CloudEvent data
+
+This extension does not support [JWE Json Serialization][jwe-json-serialization] 
+
+When using [JWE Cipher Text][jwe-ciphertext] MUST be mapped onto `data` and
+ `datacontenttype` MUST be set to `application/octet-stream`
+
+When using JWS the signed data MUST be decoded from base64url format and mapped onto
+`data` in `Binary` format where `datacontenttype` is taken from the `cty` property
+of the JOSE header, if no `cty` property exists `datacontenttype` MUST be set to `application/octet-stream`
```

Comment:
While JOSE is a required property, do we need to say that at least one of the OPTIONAL properties is required? Or is it valid to have a JOSE attribute and nothing else? Or is it "valid" but just pointless/weird? :-)